frontend/src/modules/samples/actions.ts

```diff
@@ -39,8 +39,10 @@ export const update = (id, sample) => async (dispatch, getState) => {
 export const list = (options) => async (dispatch: AppDispatch, getState) => {
     const params = { limit: 100000, ...options }
     return await dispatch(networkAction(LIST,
-        api.samples.list(params),
-        { meta: params }
+        api.samples.list(params, { abort: true }),
+        {
+            meta: params,
+        }
```

Comment:
Now that I think about it. This might be risky because there might be different React components that depends on sample list API call. So if multiple components fire this API at once, one of them will be aborted which is no good. But I don't know any page on top of my head that could cause this.

Reply:
it would have been risky, although, its pin pointed down to the samples list content page (table) even if lets say, another page would use it, the call itself will be safe until he makes a 2nd call on the same api call. its pretty safe as is :) im testing locally to see if anything changes in the app and i havent seen any changes in behavior